handler/instance_started.go

```diff
@@ -197,9 +204,21 @@ func (h *InstanceStarted) createUpdateInstanceRequest(cb cantabular.Codebook, e
 			}
 		}
 
+		id := sourceName
+		url := fmt.Sprintf("%s/code-lists/%s", h.cfg.RecipeAPIURL, sourceName)
+
+		// id and url values overwritten by codelist values.
+		// Note that cantabular codebook is sorted with exactly the same order as codelists array.
```

Comment:
This is true for my local testing. Do not merge this PR until this assumption is validated with Fran or some Cantabular documentation?